field/src/goldilocks_field.rs

```diff
@@ -317,6 +318,31 @@ impl Extendable<4> for GoldilocksField {
         [Self(0), Self(0), Self(0), Self(12587610116473453104)];
 }
 
+impl Extendable<5> for GoldilocksField {
+    type Extension = QuinticExtension<Self>;
+
+    const W: Self = Self(3);
+
+    // DTH_ROOT = W^((ORDER - 1)/5)
+    const DTH_ROOT: Self = Self(1041288259238279555);
+
+    const EXT_MULTIPLICATIVE_GROUP_GENERATOR: [Self; 5] = [
+        Self(1931274660132142120),
+        Self(1092386509894096633),
+        Self(1605533804202809407),
+        Self(14704369562396645516),
+        Self(1750907431983753016),
+    ];
```

Comment:
nit: Was this generator chosen for performance reasons? In other extension fields the generator `g` satisfies
```
g^((p^D - 1) >> F::Extension::TWO_ADICITY) = F::EXT_POWER_OF_TWO_GENERATOR
```
I can't remember now if we use this property somewhere in the code, but maybe it would be good to have it for the quintic extension too. 
Here's an example of generator that satisfies the relation:
```rust
[2899034827742553394, 13012057356839176729, 14593811582388663055, 7722900811313895436, 4557222484695340057]
```

Reply:
No, that generator was selected at random, I'm happy to use the one you suggest.

I can't find that requirement documented anywhere, so I'll write up the requirement in the `Extendable` trait.